src/Goerli-DssSpellCollateralOnboarding.sol

```diff
@@ -33,58 +33,53 @@ contract DssSpellCollateralOnboardingAction {
     //
 
     // --- Math ---
+    uint256 constant MILLION = 10 ** 6;
+
+    uint256 constant ZERO_SEVEN_FIVE_PCT_RATE = 1000000000236936036262880196;
 
     // --- DEPLOYED COLLATERAL ADDRESSES ---
+    address constant WSTETH                 = 0x6320cD32aA674d2898A68ec82e869385Fc5f7E2f;
+    address constant PIP_WSTETH             = 0x323eac5246d5BcB33d66e260E882fC9bF4B6bf41;
```

Comment:
Existing addresses should come from the Chainlog.

Reply:
Done.